utils/indigo-depict/main.c

```diff
@@ -922,11 +922,13 @@ int main(int argc, char* argv[])
         _prepare(obj, p.aromatization);
         if (p.action == ACTION_LAYOUT)
         {
-            indigoLayout(obj);
+            // indigoLayout(obj);

```

Comment:
Why is this line commented? It seems like we need to layout the molecule

Reply:
It breaks existing layout, so the line is optional.

Reply:
fixed

Reply:
are you sure? (here's another clause and the same action)

Reply:
Both indigoLayout(obj) was commented for testing purposes to prevent breaking of existing layout in the ket-files.